SocketIOClientSwift/SocketEngine.swift

```diff
@@ -495,7 +497,7 @@ public final class SocketEngine: NSObject, WebSocketDelegate {
         var n = 0
         var msg = ""
 
-        func testLength(length:String, inout n:Int) -> Bool {
+        func testLength(length: String, inout n: Int) -> Bool {
```

Comment:
Also thanks, I keep finding places where I've forgotten to remove the old style.